### DISCUSSION TITLE
How to make mini browser url input editable

### DISCUSSION THREAD

**Discussion by GNSubrahmanyam** [eclipse-theia/theia#8443] [General] [answered]
![image](https://user-images.githubusercontent.com/18217008/91651965-68b41f00-eab0-11ea-94ad-cef2b261014d.png)

**Comment by kittaakos** [comment, marked as answer]
Call the [`open`](https://github.com/eclipse-theia/theia/blob/e36195e0cd90ecb17db4e26ae29ca430d35640db/packages/mini-browser/src/browser/mini-browser-open-handler.ts#L110) method on the open handler with [`toolbar: 'show'`](https://github.com/eclipse-theia/theia/blob/e36195e0cd90ecb17db4e26ae29ca430d35640db/packages/mini-browser/src/browser/mini-browser-content.ts#L46).
